stable/ambassador/values.yaml

```diff
@@ -25,7 +25,7 @@ autoscaling:
 # namespace:
   # name: default
 
-# Additional container environment variable
+# Additional container environment variable (comma-separated list)
```

Comment:
not sure about the ` (comma-separated list)`? 
its still like this right?
```yaml
env:
  MY_ENV: "some value"
  OTHER_ENV: "value"
```

Reply:
hmm I was confused by the `{}` which made me assume that it needed to be enclosed in brackets which required variables be separated by a comma
```
env:
  {
    AMBASSADOR_ID: ambassador,
    MY_ENV: hello
  }
```
Turns out it works without the bracket in the way you described above so I went ahead and removed the bracket.

Thanks for pointing this out